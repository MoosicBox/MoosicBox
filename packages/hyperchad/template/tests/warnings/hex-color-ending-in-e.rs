use hyperchad_template::container;

fn main() {
    // This should fail at the lexer level with a helpful message
    let _ = container! {
        div color=#12e { "test" }
    };
}
