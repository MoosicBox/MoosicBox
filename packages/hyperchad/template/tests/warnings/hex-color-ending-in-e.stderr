error: expected at least one digit in exponent
 --> tests/warnings/hex-color-ending-in-e.rs:6:20
  |
6 |         div color=#12e { "test" }
  |                    ^^^
